internal/export/exportfile.go

```diff
@@ -39,12 +41,12 @@ const (
 )
 
 type ExportSigners struct {
-	SignatureInfo *database.SignatureInfo
+	SignatureInfo *model.SignatureInfo
 	Signer        crypto.Signer
 }
 
 // MarshalExportFile converts the inputs into an encoded byte array.
-func MarshalExportFile(eb *database.ExportBatch, exposures []*database.Exposure, batchNum, batchSize int, signers []ExportSigners) ([]byte, error) {
+func MarshalExportFile(eb *model.ExportBatch, exposures []*database.Exposure, batchNum, batchSize int, signers []ExportSigners) ([]byte, error) {
```

Comment:
Is this PR the precursor for a forthcoming deeper change? I had imagined from the description that we'd be breaking the link between the internal operations of exports and how data is represented in db (i.e., model objects vs DAO objects). I definitely found it weird during https://github.com/google/exposure-notifications-server/pull/353 that in order to generate an export file with no data from db I had to go through db concepts.

But unless I misread, model is just database renamed. And we've simply relocated stuff from internal/database to internal/export/database. I'd have imagined this would head towards duplicating stuff in internal/database to an internal/model and making DAO layer more explicit.

Reply:
this change is moving models to a separate place from db. Db operations for export are separate from db operations for `_other_thing_`.